element-template-generator/core/src/main/java/io/camunda/connector/generator/java/ClassBasedTemplateGenerator.java

```diff
@@ -86,6 +87,8 @@ public List<io.camunda.connector.generator.dsl.ElementTemplate> generate(
             PropertyGroup.builder()
                 .id(group.id())
                 .label(group.label())
+                .tooltip(group.tooltip().isBlank() ? null : group.tooltip())
+                .openByDefault(group.openByDefault() == Boolean.TRUE ? null : false)
```

Comment:
we return `null` when the property is `TRUE` ?

Reply:
If we don't specify anything then it is assumed to be true by default, so I'd like to avoid adding this parameter to every property in every connector 🙂